### PR TITLE
Run db init on startup

### PR DESCRIPTION
**Pull request by aochsner**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -16,5 +16,8 @@ RUN npm install --production
 # Bundle app source
 COPY src/ /usr/src/app/src/
 
+# Bundle init scripts
+COPY scripts/ /usr/src/app/scripts/
```
Comment (aochsner) [inline Dockerfile]: would it just be better to put under /src?

File: src/config/config.js [hunk 2]
```diff
@@ -18,11 +18,7 @@ const config = {
     if (this._secrets._db_url) {
       return this._secrets._db_url;
     }
-    const hostname = cls.getNamespace('hapi-request').get('host');
-    if (hostname.startsWith('localhost')) {
-      return 'http://localhost:5984';
-    }
-    return 'http://couchdb.riglet:5984';
+    return 'http://localhost:5984';
```
Comment (aochsner) [inline src/config/config.js]: i can't remember why we bother w/ this magic...the Dockerfile now always sets TWIG_API_DB_URL

File: src/server.js [hunk 3]
```diff
@@ -12,6 +12,7 @@ const helpers = require('./server.helpers');
 const v2 = require('./api/v2');
 const config = require('./config');
 const semver = require('semver');
+require('../scripts/init-new-db');
```
Comment (aochsner) [inline src/server.js]: too sneaky?

File: Dockerfile (hunk 1 above)
Comment (BenAychh) [reply Dockerfile]: I'm ok with that. Especially since people won't need to do anything with it anymore.

File: src/config/config.js (hunk 2 above)
Comment (BenAychh) [reply src/config/config.js]: I didn't see a reason either when I was refactoring this. Glad it is gone.

File: src/server.js (hunk 3 above)
Comment (BenAychh) [reply src/server.js]: Maybe a little bit? I can see this getting lost and people having to figure it out.

Should we create an `initDb()` function then do `const { initDb } = require('../scripts/init-new-db')` and explicitly call it?

**Pull request merged**